test/codegen/golden/expected/ecommerce/app/routers/orders.py

```
@@ -0,0 +1,106 @@
+async def create_draft_order(
+    body: OrderCreate,
+    session: AsyncSession = Depends(get_session),
+) -> OrderRead:
+    svc = OrderService(session)
+    return await svc.create_draft_order(body)
+
+@router.post("/orders/{order_id}/items", status_code=201)
+async def add_line_item(
+    order_id: str,
+    body: OrderUpdate,
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)